hera_sim/visibilities/cli.py

```diff
@@ -167,7 +167,7 @@ def run_vis_sim(args):
         comm.Barrier()
         sys.exit(0)
 
-    if myid == 0:
+    if args.run_auto_check:
```

Comment:
The auto check should still be done by rank 0 process, right?

Reply:
Well, all other processes exit in the statement above, so only rank 0 does anything beyond that point. I thought it was nicer to not have the extra unnecessary indent.

Reply:
Ah yes, I see now. The rank checking is not necessary then.